backoffice/src/components/backoffice/common/Header.jsx

```diff
@@ -28,10 +30,32 @@ Link.propTypes = {
 
 export default class Header extends React.Component {
 
+    state = {
+
+    }
+
     static propTypes = {
         onLogout: PropTypes.func.isRequired,
     };
 
+    componentDidMount() {
+        this.search();
+    }
+
+    search = (options = {}) => {
+        return new Promise(resolve => {
+            this.setState({ loading: !options.silent }, async () => {
```

Comment:
je sais pas si c'est possible de faire un truc comme ça ?
this.setState(prevState => ({ loading: !prevState.options.silent }), async () ...